LicensePlist.podspec.tmp

```diff
@@ -0,0 +1,9 @@
+Pod::Spec.new do |s|
+  s.name            = 'LicensePlist'
+  s.version         = 'LATEST_RELEASE_VERSION_NUMBER'
+  s.summary         = 'A license list generator of all your dependencies for iOS applications'
+  s.homepage        = 'https://github.com/mono0926/LicensePlist'
+  s.license         = { :type => 'MIT', :file => 'LICENSE' }
+  s.author          = 'Masayuki Ono'
+  s.source          = { :http => '#{s.homepage}/releases/download/#{s.version}/portable_licenseplist.zip' }
+end
```

Comment:
In order to deliver CocoaPods updates in release.sh, I wanted to be able to set the version number externally.

Therefore, I thought that create a new podspec dynamicary, and replace version number.

> LicensePlist.podspec.tmp
> ```ruby
> s.version = 'LATEST_RELEASE_VERSION_NUMBER'
> ```

> release.sh
> ```bash
> cat "$podspec_name.tmp" | sed s/LATEST_RELEASE_VERSION_NUMBER/$tag/ > "$podspec_name"
> ```
If you have any other good ideas please let me know.